linera-service/src/storage.rs

```diff
@@ -318,6 +318,30 @@ impl FullStorageConfig {
             }
         }
     }
+
+    /// List all the tables of the database
+    pub async fn list_tables(self) -> Result<Vec<String>, ViewError> {
+        match self {
+            FullStorageConfig::Memory(_) => Err(ViewError::ContextError {
+                backend: "memory".to_string(),
+                error: "delete_single does not make sense for memory storage".to_string(),
+            }),
+            #[cfg(feature = "rocksdb")]
+            FullStorageConfig::RocksDb(_) => {
+                panic!("Currently the list_tables is not supported for RocksDb");
```

Comment:
"list_tables is not supported for the rocksdb storage"

Please let's be consistent: either the Rocksdb and Memory cases both return an error or they both panic.

Reply:
Ok, I thought it was different contextes. Because memory will never have a "list_table", while this is conceivable for `RocksDb`.

So, I will use `Err`.

Reply:
I see. I guess it's just not very nice to tell the user that what they wrote "doesn't make sense".